recipe/bld.bat

```diff
@@ -1,16 +1,22 @@
 mkdir build
 cd build
 
+set VERBOSE=1
+
 cmake -G "NMake Makefiles" ^
       -D CMAKE_BUILD_TYPE=Release ^
```

Comment:
would https://github.com/conda-forge/h3-feedstock/commit/5d920f6389320cbdf1fb3e02e0fd8cef61a48fbb these changes fix it?